services/venice-server/src/main/java/com/linkedin/venice/listener/grpc/handlers/GrpcOutboundResponseHandler.java

```diff
@@ -53,7 +53,7 @@ public void processRequest(GrpcRequestContext ctx) {
     ctx.setError();
     statsContext.setResponseStatus(NOT_FOUND);
     veniceServerResponseBuilder.setData(ByteString.EMPTY);
-    veniceServerResponseBuilder.setErrorCode(GrpcErrorCodes.KEY_NOT_FOUND);
+    veniceServerResponseBuilder.setErrorCode(VeniceReadResponseStatus.KEY_NOT_FOUND);
```

Comment:
Will this break anything? The old `GrpcErrorCodes.KEY_NOT_FOUND` has a value of 101 and the new value is now -420.

Reply:
There are no production use cases yet, so this breaking change won't have any impact.

Reply:
It might cause confusion later when we try to productionize GRPC, can we add a new static field in `VeniceReadResponseStatus` called `GRPC_KEY_NOT_FOUND` and set its value to 101?

Reply:
>called GRPC_KEY_NOT_FOUND and set its value to 101?

QQ: Why do we need to call out gRPC in the var name?

Reply:
You are right, we don't need to have separate error code for `KEY_NOT_FOUND` since we own the handling for this error end to end so as long as it doesn't conflict with any existing error code we should be fine.